samples/RXPTest/src/Tests/TextInputInteractiveTest.tsx

```diff
@@ -235,14 +235,16 @@ class TextInputView extends RX.Component<RX.CommonProps, TextInputViewState> {
                     onSubmitEditing={ this._onSubmitEditingTest6 }
                     multiline={ true }
                 />
-                <RX.ScrollView style={ _styles.eventHistoryScrollView }>
-                    <RX.Text style={ _styles.eventHistoryText }>
-                        { this.state.test6EventHistory.length ?
-                            this.state.test6EventHistory.join('\n') :
-                            'Event history will appear here'
-                        }
-                    </RX.Text>
-                </RX.ScrollView>
+                <RX.View style={ _styles.eventHistoryScrollView }>
```

Comment:
I couldn't get the ScrollView to show up on Web without adding this container.  Didn't have time to dig into why.  Should we create an issue to investigate?